ush/bash_utils/process_args.sh

```diff
@@ -310,7 +310,13 @@ but the element with index i=${i} is empty:
   for arg_val_pair in "${@:2}"; do
 
     arg_name=$(echo ${arg_val_pair} | cut -f1 -d=)
-    arg_value=$(echo ${arg_val_pair} | cut -f2 -d=)
+# The following method with sed also works.
+#    arg_name=$(echo ${arg_val_pair} | $SED -n -r "s|^([^=]*)(=)(.*)|\1|p")
```

Comment:
Since arg_value is moving to sed, should we go ahead and use the arg_name sed version as well?  Similarly, if both sed versions are used, then the cut versions should probably be removed, rather than commented out.

Reply:
@MichaelLueken I found a simpler bash-based solution that doesn't need `cut` or `sed`.  Will push it soon.